hokusai/production.yml

```diff
@@ -107,7 +107,7 @@ spec:
   type: LoadBalancer
 
 ---
-apiVersion: v1
```

Comment:
Ah yes @ansor4 - a bit confusing as these deployment objects go through different API versions - but you can look up the spec for our k8s version (1.10.4) here https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#cronjob-v1beta1-batch